packages/ast/src/nodes/ArrayElement.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * @module ast/nodes
+ */
+
+/**
+ * Copyright 2018 Alexandru RADOVICI
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+
+import { Type } from '../types';
+import { Expression } from './Expression';
+import { NodeID, ParentNode } from '../util//Node';
+import { Node } from "../util/Node";
+import { ASTError } from '../errors';
+import { iJSON } from '../util/JSON';
```

Comment:
```suggestion
import { Type } from '../types';
import { Expression } from './Expression';
import { NodeID, ParentNode } from '@easycompiler/util/';
import { Node } from "@easycompiler/util";
import { ASTError } from '@easycompiler/errors';
import { iJSON } from '@easycompiler/util';
```

I would export all the items in the root of the package.